release/apt-repo/Earthfile

```diff
@@ -73,28 +67,14 @@ Expire-Date: 0
     SAVE ARTIFACT earthly-pgp-public.pgp AS LOCAL earthly-pgp-public.pgp
     SAVE ARTIFACT earthly-pgp-private.pgp AS LOCAL earthly-pgp-private.pgp
 
-
-aws-base:
-    FROM alpine:latest
-    RUN apk add --update --no-cache curl unzip
-    RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.0.30.zip" -o "awscliv2.zip"
-    RUN unzip awscliv2.zip
-    RUN ./aws/install
-    ENV PATH=$PATH:/usr/local/aws-cli/v2/bin/
-
-aws:
-    FROM amazon/aws-cli
-    RUN mkdir -p ~/.aws && echo "[profile developer]
-    role_arn = arn:aws:iam::404851345508:role/developer
-    source_profile = default" > ~/.aws/config
-    ENV AWS_PROFILE=developer
-
 download:
-    FROM +aws
+    FROM ../common-repo+aws
     ARG --required S3_BUCKET
-    RUN env | grep dev
     RUN --no-cache \
+        --secret MFA_ARN=+secrets/user/earthly-technologies/aws/mfa-arn \
+        --secret MFA_KEY=+secrets/user/earthly-technologies/aws/mfa-key \
         --mount type=secret,id=+secrets/user/earthly-technologies/aws/credentials,target=/root/.aws/credentials \
```

Comment:
These credentials should _also_ require an MFA session? Are you MFA locally and re-setting your credential files?

Reply:
`/root/.aws/credentials` never gets overwritten, nor does it contain any MFA-related session data.

`aws sts assume-role` is only called once, and if I'm not mistaken, uses both the user's regular `credentials`, which contain `aws_access_key_id` and `aws_secret_access_key`, and the TOTP token value, which is passed to `--token-code`.